feature-libs/asm/core/store/reducers/asm-ui.reducer.ts

```diff
@@ -1,17 +1,18 @@
+import { Action } from '@ngrx/store';
 import { AsmUi } from '../../models/asm.models';
 import { AsmActions } from '../actions/index';
 
 export const initialState: AsmUi = <AsmUi>{ collapsed: false };
 
 export function reducer(
-  state = initialState,
-  action: AsmActions.AsmUiAction
+  state: AsmUi | undefined = initialState,
```

Comment:
Do we need this change at all?

Reply:
Good catch, you're right.  Fixed.
Again, likely a leftover from some back and forth in the solutions applied.  I'll keep an eye out for those next time.